proto-compiler/Cargo.lock

```diff
@@ -0,0 +1,1184 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Not sure this is the right PR to add the Cargo.lock for proto-compiler crate.

Reply:
My mistake!